applications/plugins/Geomagic/src/Geomagic/GeomagicDriver.cpp

```diff
@@ -173,7 +175,7 @@ GeomagicDriver::GeomagicDriver()
     this->f_listening.setValue(true);
     m_forceFeedback = nullptr;
     m_GeomagicVisualModel = std::make_unique<GeomagicVisualModel>();
-    sofa::core::objectmodel::BaseObject::d_componentState.setValue(sofa::core::objectmodel::ComponentState::Loading);
+    sofa::core::objectmodel::BaseObject::d_componentstate.setValue(sofa::core::objectmodel::ComponentState::Loading);
```

Comment:
Sorry, too many changes in my dev branch. 
I'll try to finish the PR regarding that changes for the next meeting to avoid moving backward here.